system/security/securesocket/securesocket.cpp

```diff
@@ -140,7 +140,7 @@ class CSecureSocket : implements ISecureSocket, public CInterface
     bool        m_isSecure;
     size32_t    nextblocksize;
```

Comment:
I think nextblocksize and blockflags are also not initialized by constructor

Reply:
Good point.  I wrongly assumed Coverity listed _all_ uninitialized members and didn't check.  I will add these.
Not sure if nextblocksize should be 0 or UINIT_MAX.  It is 0 in the CSocket ctor but set to UINIT_MAX in the prerequisite set_block_mode() call.

Reply:
Updated code, made nextblocksize 0 to match CSocket ctor, also initialized blocktimeoutms in jsocket.
Pushed commit.